docs/managing-findings.md

```diff
@@ -0,0 +1,108 @@
+# Managing findings
+
+[TOC]
+
+# Findings
+
+A finding is the core unit of Semgrep's analysis. Findings are generated when a Semgrep rule matches a piece of code. After matching, a finding can make its way through 3 parts of the Semgrep ecosystem: [`semgrep`](https://github.com/returntocorp/semgrep), [`semgrep-action`](https://github.com/returntocorp/semgrep-action), and [semgrep.dev](https://semgrep.dev/).
+
+## Semgrep CLI
+
+Semgrep CLI findings are a specific rule matching a piece of code. Multiple rules can match the same piece of code, even if they are effectively the same rule. For example, consider the following rule and code snippet:
+
+```
+rules:
+- id: finding-test
+  pattern: $X == $X
+  message: Finding test 1
+  languages: [python]
+  severity: WARNING
+- id: finding-test
+  pattern: $X == $X
+  message: Finding test 2
+  languages: [python]
+  severity: WARNING
+```
+
+```
+print(1 == 1)
+```
+
+Running Semgrep CLI produces the following findings:
+
+```
+$ semgrep --quiet --config test.yaml test.py
+test.py
+severity:warning rule:finding-test: Finding test 1
+1:print(1 == 1)
+--------------------------------------------------------------------------------
+severity:warning rule:finding-test: Finding test 2
+1:print(1 == 1)
+```
+
+*For more information on writing rules, see [Rule syntax](writing-rules/rule-syntax.md).*
+
+## Semgrep Action
+
+Semgrep Action builds on Semgrep CLI findings to track the lifetime of an individual finding.
```

Comment:
How about adding a tiny bit about Semgrep Action, like “designed to continuously scan commits and builds”?

```suggestion
Semgrep Action, designed to continuously scan commits and builds, builds on Semgrep CLI findings to track the lifetime of an individual finding.
```

Reply:
```suggestion
Semgrep Action, designed to continuously scan commits and builds, improves on Semgrep CLI findings to track the lifetime of an individual finding.
```